pkg/cmd/server/apis/config/v1/register.go

```diff
@@ -9,27 +9,60 @@ import (
 	"github.com/openshift/origin/pkg/cmd/server/apis/config"
 )
 
-const GroupName = ""
+var (
+	GroupName    = "config.openshift.io"
```

Comment:
oh my.  We're going for it?  Any chance we can defer this aspect.  I suspect we'll split this group and keeping only the legacy name ensures downgrades won't instantly fail.

Reply:
I think except cluster up nothing in code use this for now (and even that part use the legacy types on backgroud...)... so I can remove this from this PR (I also think this is not the right PR to introduce new group ;-)